packages/components/src/Autocomplete/Menu.tsx

```diff
@@ -38,7 +44,17 @@ export function Menu({
       {options.map((option, index) => {
         const optionClass = classnames(styles.option, {
           [styles.active]: index === highlightedIndex,
+          [styles.separator]: addSeparators,
         });
+        if (isGroup(option)) {
+          return (
+            <div key={option.label} className={styles.heading}>
+              <Typography element="h5" fontWeight="bold">
```

Comment:
Can we use a [`Heading`](https://atlantis.getjobber.com/packages-components-src-heading-heading) here instead?